tap_paycor/__init__.py

```diff
@@ -52,6 +52,32 @@ def discover():
     return Catalog(streams)
 
 
+def refresh_token(args):
+    root_url = 'https://apis.paycor.com'
+    config = args.config
+    key = os.environ['PAYCOR_API_SUBSCRIPTION_KEY']
+    url = f"{root_url}/sts/v1/common/token?subscription-key={key}"
+    headers = { "content-type": "application/x-www-form-urlencoded" }
+    data = {
+        "grant_type": "refresh_token",
+        "refresh_token": os.environ["PAYCOR_REFRESH_TOKEN"],
+        "client_id": os.environ["PAYCOR_CLIENT_ID"],
+        "client_secret": os.environ["PAYCOR_CLIENT_SECRET"],
+    }
+    response = requests.post(url, data=data, headers=headers)
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as error:
+        # todo need someway to log and notify. DAG is dead until this is fixed
+        raise error
+
+    # update args with new config
+    args.config = response.json()
```

Comment:
the response won't come back with a complete config.json, we want to just patch the access and refresh tokens from the response and preserve the rest of what originally came in via config.json

Reply:
Fixed in [c0827db](https://github.com/JarvusInnovations/tap-paycor/pull/7/commits/c0827db59a9928f3eebc3af30b5c2fe4cc456410).